### DISCUSSION TITLE
BOM Item default sorting order

### DISCUSSION THREAD

**Discussion by LastEdit** [inventree/InvenTree#7069] [Q&A] [answered]
Is there a way to display the BOM Items in a BOM report or on the site in a specific consistent order, basically assigning each item a display order? Right now it seems like it might be related to when the part was added to the BOM Item table, but I'm not 100% sure. Sometimes when changing a part, it will go to the bottom of the list.

I know you can sort it by the different columns available (alpha numeric), but I am looking for a sort order that is irrespective of that.

**Comment by SchrodingersGat** [comment, marked as answer]
@LastEdit you would have to anchor the ordering to a column in the BomItem table which already exists - such as the part ID, name, etc. We do not have a way to assign an arbitrary "ordering" to the BomItems in a way that you describe

**Reply by wolflu05** [reply]
This sounds like a nice usecase for custom metadata and plugins. Maybe he can use a custom panel then to display an orderable list and save that to metadata, and then access that information on report generation.
